src/replays_utils.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const { DemoParser } = require('sdfz-demo-parser');
+
+// Parse a replay and create the needed info structure. If there is a cache
+// file for this replay, use it.
+async function parseReplay(springPath, replayPath, exitIfCacheExists=true) {
```

Comment:
please avoid default parameters (personal dislike of mine), as they can be a source of bugs (API semantics changing in an invisible way);

also this is possibly a bug, if you return null when nothing is specified, then the `ReadReplayInfo` command in `replay_handler`:
```
const info = parseReplay(springPlatform.writePath, command.relativePath);
```
would return null for cached replays - probably not what you want!